rustls/src/client/builder.rs

```diff
@@ -76,6 +76,22 @@ impl ConfigBuilder<ClientConfig, WantsClientCert> {
         Ok(self.with_client_cert_resolver(Arc::new(resolver)))
     }
 
+    /// Sets a single certificate chain and matching private key for use
+    /// in client authentication.
+    ///
+    /// `cert_chain` is a vector of DER-encoded certificates.
+    /// `key_der` is a DER-encoded RSA, ECDSA, or Ed25519 private key.
+    ///
+    /// This function fails if `key_der` is invalid.
+    #[deprecated(since = "0.21.3", note = "Use `with_client_auth_cert` instead")]
```

Comment:
I marked this as "since 0.21.3" but perhaps it should be the _next_ release? 0.22.0?

Reply:
The first release in which it will be deprecated might be 0.21.4, then we can remove it in 0.22.

Reply:
Switched to 0.21.4 and will pull this into https://github.com/rustls/rustls/pull/1348